### ISSUE TITLE
Include external libraries in final bundle

### ISSUE THREAD

**Issue by rojasjandro89** [ng-packagr/ng-packagr#1231]
## Type of Issue

```
[ ] Bug Report
[x ] Feature Request
```

## Description

Need the ability to include specified dependencies in the bundle. I have an app that will import these bundles dynamically at runtime and there's no way to know the dependencies of these bundles until they are loaded, at that point resolving them is not possible. 

#### Expected Behaviour

I should be able to include specified dependencies in the final bundle (even when this is considered a bad practice)

**Comment by gs-nchakravarthi** [comment]
Yes, Even there are few use-cases where i wanted to achieve this.

I wanted to bundle a dependency developed by other team of mine, so as to make sure the consumer of my library need not import this external dependency. 

Mostly, dependencies like @angular/lodash/rxjs etc., will be already bundled by the consumer of my library. But there are some dependencies which needed to be bundled mandatorily with the library.

**Comment by alan-agius4** [comment]
Duplicate of https://github.com/ng-packagr/ng-packagr/issues/1106

**Comment by rojasjandro89** [comment]
Thanks for the quick reply @alan-agius4. I followed your instructions on #1106 and even it certainly includes the dependencies in the bundle, but it includes them all!! Even if I do `"bundledDependencies": ["moment"]`

It will bundle everything not declared in peerDependencies. Am I missing something? This is the package.json:
```
{
  "name": "dummyModule",
  "version": "0.0.1",
  "bundledDependencies": [
    "moment"
  ],
  "peerDependencies": {
    "@angular/common": "^7.2.0",
    "@angular/core": "^7.2.0"
  }
}
```
I'm using the angular CLI to export a very simplistic angular module that happens to use momentjs

**Comment by rojasjandro89** [comment]
Is there any progress on this topic?

**Comment by briancodes** [comment]
@rojasjandro89 Were you able to get this resolved? There's not much documentation on it

**Comment by pacocom** [comment]
@rojasjandro89 @briancodes I have the same problem! Can you help me?
I need higcharts and mapbox outside main-es2015.js
![image](https://user-images.githubusercontent.com/7735329/73675427-adaadb00-46b2-11ea-9ccb-c1c86b43709b.png)

Thanks.

**Comment by briancodes** [comment]
@pacocom I added `bundledDependencies` and `dependencies` to the `package.json` in the `lib` folder (whatever was in bundledDependencies needed to be in dependencies also). I then built the lib, and from the `dist/my-lib` folder ran `npm install` and `npm pack` - this created the `.tgz` file. Then `npm publish ./dist/my-lib/my-lib.1.2.0.tgz`. These commands are from memory here, but pretty sure that's it
